packages/perseus/src/styles/articles.less

```diff
@@ -121,7 +121,6 @@
         font-size: 14px;
         line-height: 19px;
         margin: 16px auto 42px;
-        max-width: 455px;
```

Comment:
**thought**: I agree with removing this, but I always wonder where some of these seemingly arbitrary rules came from. :)